### PR TITLE
[OBSDEF-6252] Enable stdout logs handling in vsphere

### PR DESCRIPTION
**Pull request by AnastasiaShemyakinskaya**
## Purpose
[OBSDEF-6252](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-6252)

* Add RSYSLOG_ENABLE and RSYSLOG_STDOUT_ENABLE  variables in Makefile to pass it in `make package` for logging-injector and objectscale-vsphere

## PR checklist
- [ ] make test passed
- [ ] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -286,6 +290,9 @@ create-logging-injector-app: create-temp-package
 	--set global.registry=${REGISTRY} \
 	--set global.registrySecret=${REGISTRYSECRET} \
 	--set global.objectscale_release_name=objectscale-manager \
+	--set global.objectscale_release_name=objectscale-manager \
```
Comment (vazhem) [inline Makefile:293]: duplicated line
--set global.objectscale_release_name=objectscale-manager

File: Makefile [hunk 2]
```diff
@@ -51,6 +51,8 @@ HELM_DECKS_ARGS      = # --set image.tag=${YOUR_VERSION_HERE}
 HELM_KAHM_ARGS       = # --set image.tag=${YOUR_VERSION_HERE}
 HELM_DECKS_SUPPORT_STORE_ARGS      = # --set decks-support-store.image.tag=${YOUR_VERSION_HERE}
 SED_INPLACE         := -i
+RSYSLOG_ENABLE      := false
```
Comment (vazhem) [inline Makefile:54]: why do you need new flag RSYSLOG_ENABLE? (it is already defined arg in monitoring charts, so maybe not much need to have it in this makefile)
we have rsyslog enabled by default, so if you want to introduce it as a flag here it should be set to 'true', not 'false'

File: Makefile [hunk 3]
```diff
@@ -51,6 +51,8 @@ HELM_DECKS_ARGS      = # --set image.tag=${YOUR_VERSION_HERE}
...
+RSYSLOG_STDOUT_ENABLE   := false
```
Comment (vazhem) [inline Makefile:55]: I would rename the flag to something more readable like 'ENABLE_STDOUT_LOGS_COLLECTION'

File: Makefile (hunk 2 above)
Comment (victor-polyansky) [reply Makefile:54]: My bad, I missed that it shouldn't be set in Makefile.

@AnastasiaShemyakinskaya don't add it here, just change to `true` in logging-injector chart in monitoring (https://eos2git.cec.lab.emc.com/ECS/monitoring/blob/master/k8s/charts/logging-injector/values.yaml#L29). In objectscale-vsphere it's already set to true.

**Review by vazhem — APPROVED** [review]

**Review by victor-polyansky — APPROVED** [review]

**Pull request merged**